lib/fauna/query.rb

```diff
@@ -91,7 +105,8 @@ def var(name)
     #
     # Reference: {FaunaDB Basic Forms}[https://faunadb.com/documentation/queries#basic_forms]
     def if_(condition, then_, else_)
-      { if: condition, then: then_, else: else_ }
+      # rubocop:disable Style/HashSyntax
```

Comment:
Why the suppression and use of hash rockets here and in the next method, but not elsewhere?